packages/block-editor/src/components/block-alignment-control/test/index.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import BlockAlignmentUI from '../ui';
+
+it( 'should call onChange with undefined, when the control is already active', () => {
```

Comment:
Regarding the `when the control is already active` part, do we want to test that `onChange` is called with `undefined` when tapping the currently active alignment value? If this is the case, on line `18` we should get the left alignment button, right?

Reply:
Great catch. The odd pairing of relying upon the default left alignment and pressing "None", meant this test passed even though it was not truly the "selected" option. I addressed this in https://github.com/WordPress/gutenberg/pull/35191/commits/bcd16d620af229b66d2f3dd3b22f293552005144 by shifting the test right alignment to be more explicit.

Reply:
Thanks for addressing it ❤️ !

I'm thinking that the previous code was covering the new logic of the `None` option. Maybe it's worth it to include that test case (having a selected alignment option and select `None`), wdyt?